controllers/villains.js

```diff
@@ -0,0 +1,41 @@
+const models = require('../models')
+
+
+const serverSetup = (req, res) => {
+  return res.send('Get ready to meet our villains!')
+}
+
+const displayAll = async (req, res) => {
+  const villains = await models.Villains.findAll()
+
+  return res.send(villains)
+}
+
+const returnBySlug = async (req, res) => {
+  const { slug } = req.params
+  const returnBySlug = await models.Villains.findAll({ where: { slug } })
+
+  return res.send(returnBySlug)
```

Comment:
what happens if nothing is found... you send blank array?

Not 404

minor but please remember in real api 404 should be returned if nothing is found by specific item